astropy/io/fits/tests/test_table.py

```diff
@@ -1049,14 +1049,13 @@ def test_assign_multiple_rows_to_table(self):
         assert tbhdu2.columns.columns[2].array[4] == ''
         assert (tbhdu2.columns.columns[3].array[4] ==
                 np.array([1., 2., 3., 4., 5.], dtype=np.float32)).all()
-        assert tbhdu2.columns.columns[4].array[4] == False
-
+        assert bool(tbhdu2.columns.columns[4].array[4]) is False
```

Comment:
This seems odd; if `array` elements are not `bool` already, I would just test with
```
assert not tbhdu2.columns.columns[4].array[4]
```